dluhc-component-library/src/components/timetablePage/TimetablePage.tsx

```diff
@@ -2,42 +2,54 @@ import { useEffect, useState } from "preact/hooks";
 import csvToJson from "csvtojson";
 import { loadCSV, loadJson } from "../../utils";
 import Timetable from "./Timetable";
-import { TimetableStage } from "./types";
+import { TimetableHeader, TimetableStage } from "./types";
 import AccordionDropdown from "./AccordionDropdown";
 
 interface TimetablePageProps {
   filepath: string;
+  headersFilepath: string;
 }
 
-const TimetablePage = ({ filepath }: TimetablePageProps) => {
+const TimetablePage = ({ filepath, headersFilepath }: TimetablePageProps) => {
```

Comment:
We should probably rename the "filepath" prop now we have another one make it obvious what its used for. Something along the lines of "timetableFilepath" or "stagesFilepath".